src/javascript/JContent/actions/publishAction.jsx

```diff
@@ -69,9 +69,8 @@ export const PublishActionComponent = ({context, render: Render, loading: Loadin
         displayName: _.escape(ellipsizeText(res.node.displayName, 40)),
         language: res.node.site ? _.escape(uppercaseFirst(getLanguageLabel(res.node.site.languages, language).displayName)) : null
     } : {
-        displayName: t('jcontent:label.contentManager.selection.itemsSelected', {count: context.paths.length}),
-        language: res.nodes[0].site ? _.escape(uppercaseFirst(getLanguageLabel(res.nodes[0].site.languages, language).displayName)) : null,
-        multipleSelectionDisplayName: t('jcontent:label.contentManager.selection.items', {count: context.paths.length})
+        displayName: t('jcontent:label.contentManager.selection.items', {count: context.paths.length}),
```

Comment:
you still have some stuff in getButtonRenderer that can be cleaned

Reply:
yep